### PR TITLE
Cbj 20160223 latest benchmark

### PR DESCRIPTION
**Pull request by cjacoby**
- Function for loading the latest benchmark
- started jupyter notebook template for viz

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
         'pescador >= 0.1.2',
         'pytest',
         'pytest-benchmark',
+        'pandas >= 0.17.0'
```
Comment (ejhumphrey) [inline setup.py:36]: http://www.wcvb.com/national/aww-giant-panda-takes-a-tumble-in-the-snow/38121224

File: benchparse.py [hunk 2]
```diff
@@ -19,16 +19,22 @@
 parameters.
```
Comment (ejhumphrey) [inline benchparse.py:19]: is it worth moving this module to `minibench/parse.py` or whathaveyou?

**Comment by ejhumphrey** [comment]
lgtmyo

**Comment by ejhumphrey** [comment]
...but i'll leave it to you to mergeymerge

File: benchparse.py (hunk 2 above)
Comment (cjacoby) [inline benchparse.py:19]: I couldn't decide. It didn't quite feel like it belonged there, but I could go either way.

Comment (cjacoby) [inline benchparse.py:19]: ... leavin' it for now, you can move it if it makes you happy.

**Pull request merged**